src/main/scala/com/exasol/cloudetl/bucket/AzureBlobBucket.scala

```diff
@@ -86,7 +87,14 @@ final case class AzureBlobBucket(path: String, params: StorageProperties) extend
   private[this] def regexParsePath(path: String): AccountAndContainer = path match {
     case AZURE_BLOB_PATH_REGEX(containerName, accountName, _) =>
       AccountAndContainer(accountName, containerName)
-    case _ => throw new BucketValidationException(s"Invalid Azure blob wasb(s) path: $path!")
+    case _ =>
+      throw new BucketValidationException(
+        ExaError
+          .messageBuilder("E-CSE-19")
+          .message("Azure blob wasb(s) path {{PATH}} is not valid.")
```

Comment:
is wasb(s) a domain specific term?

Reply:
Yes, it is path scheme for Azure blob storage. Maybe I can reword it and remove this term.